nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise 'NotImplementedError'
```

Comment:
- [ ] [_Optional_] Adding a custom message is always a good idea with default errors so you can distinguish easily where the error occurred and why.
``` ruby
 raise 'NotImplementedError', "#{self.class} has not implemented method '#{__method__}'"
```